<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="Papeles">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>301</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Info_button">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>260</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="Tiempo_Label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tiempo:</string>
    </property>
   </widget>
   <widget class="QListWidget" name="Planilla_Control">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>11</y>
      <width>431</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>401</y>
      <width>431</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Aprobado_Button">
       <property name="text">
        <string>APROBADO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Denegado_Button">
       <property name="text">
        <string>DENEGADO</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>400</y>
      <width>254</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="Guardar_Button">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Cargar_Button">
       <property name="text">
        <string>Cargar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Jugar_Button">
       <property name="text">
        <string>Jugar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nivel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="nivel_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha:</string>
    </property>
   </widget>
   <widget class="QLabel" name="fecha_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>06/07/2023</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Personas Restantes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="personasR_Label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>350</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
